micropython/lora/lora-sx126x/lora/sx126x.py

```diff
@@ -152,7 +152,8 @@ def __init__(
             dio3_tcxo_start_time_us if dio3_tcxo_millivolts else 0
         )
 
-        self._buf = bytearray(9)  # shared buffer for commands
+        self._buf = bytearray(9)  # shared buffer for commands, access through _buf_view
+        self._buf_view = memoryview(self._buf) 
```

Comment:
The memoryview will keep the buffer alive by itself, so I think this can be shrunk down to

```py
self._buf = memoryview(bytearray(9))  # memoryview into shared buffer for commands
```

Reply:
Gladly - didn't know whether the gc would cause issues in a case like this. Good to know, thanks!